sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -569,3 +570,29 @@ public abstract partial class ObjectSerializer
         public virtual System.Threading.Tasks.ValueTask<System.BinaryData> SerializeAsync(object? value, System.Type? inputType = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
     }
 }
+namespace Azure.Messaging
+{
+    [System.Text.Json.Serialization.JsonConverterAttribute(typeof(Azure.Messaging.CloudEventConverter))]
+    public partial class CloudEvent
```

Comment:
I disabled nullable references for this type since Azure.Core has them enabled and we want to avoid exposing nullable refs in APIs.

Reply:
>we want to avoid exposing nullable refs in APIs.

Why?

Reply:
versioning concerns - we don't have tooling to detect breaks in nullability.

Reply:
We exclude core from that rule and are being very careful instead.

Reply:
Well, I don't have a strong preference here. I preferred removing it for consistency with the other libraries, but am open to adding it back.
/cc @KrzysztofCwalina @tg-msft

Reply:
We already use nullable references in Azure.Core.dll (only). We don't want to use this feature in other dlls/packages.

Reply:
Which reminds me: BCL APIs have the annotations. Did we annotate System.Memory.Data?

Reply:
yes, we did. I can re-enable for these two files.